lib/paper_trail/reifier.rb

```diff
@@ -0,0 +1,256 @@
+module PaperTrail
+
+  # Given a version record and some options, builds a new model object.
+  # @api private
```

Comment:
`@api private` indicates a private part of a library's API.

I've seen it, and its inverse `@api public`, used in major ruby libraries like rspec (https://github.com/rspec/rspec-expectations/blob/14faeab88f319ac0c2e4d793ec02c2b69eb52a5c/lib/rspec/matchers/composable.rb#L9).  I think it comes from javadoc originally.